vowpalwabbit/gd.cc

```diff
@@ -253,7 +255,7 @@ inline void audit_feature(audit_results& dat, const float ft_weight, const uint6
     tempstream << ':' << (index >> stride_shift) << ':' << ft_weight << ':'
                << trunc_weight(weights[index], (float)dat.all.sd->gravity) * (float)dat.all.sd->contraction;
 
-    if (dat.all.adaptive)
+    if (stride_shift > 0) // adaptive
```

Comment:
This does not seem correct, because the stride_shift can be > 0 without adaptive being on.   Basically, either --normalized or --adaptive can cause this.